spec/components/record_actions/view_spec.rb

```diff
@@ -64,4 +64,20 @@
       include_examples "no actions"
     end
   end
+
+  context "when course date is in the future" do
+    let(:trainee) { build(:trainee, :submitted_for_trn, course_start_date: Date.parse("01/01/2099")) }
```

Comment:
Instead of `Date.parse("01/01/2099")`, you could use say `1.day.from_now` or `x.days.from_now` - your pick.

Reply:
thanks